src/jit/jitconfigvalues.h

```diff
@@ -218,6 +218,10 @@ CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 0)
 CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 1)
 #endif // !defined(DEBUG) && !defined(_DEBUG)
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+CONFIG_INTEGER(JitMinOptsTrackGCrefs, W("JitMinOptsTrackGCrefs"), 0) // Allow GC roots to be tracked w/MinOpts
```

Comment:
Missed your second comment until now; sorry about that. OK, I will change this as you suggest, but it still leaves the issue of the assert in vm\gcinfodecode.cpp - assuming it can see the above flag (the whole CLR vs. JIT config flags thing is still a mystery to me), it will need to test the flag when decoding GC info (at runtime) or we just weaken the decoding assert for every config / all the time? Neither approach seems ideal to me, but there's probably a way to do this better that I'm not seeing; please bear with me.

And, just to make absolutely sure - with this newest change, the new code will always be compiled in and we'll use a runtime test of (pseudo-code) config.JitMinOptsTrackGCrefs in the various spots in the *.cpp files instead of switching the code out completely (using the current "#ifdef AMD64 && blah blah"); correct?